docs/typescript.mdx

```diff
@@ -35,47 +48,37 @@ const titleStyle = css({
 });
 ```
 
-To make the css prop work with pure TypeScript (without babel plugin) you need to add `/** @jsx jsx */` at the top of every file that is using the css prop:
-
-```tsx
-/** @jsx jsx */
-import { jsx } from '@emotion/react'
-
-<div css={{ background: 'black' }} />
-```
-
-As a result you may be not able to use react fragment shorthand syntax - `<></>`, but still you can use `<Fragment></Fragment>`.
-This is a limitation of the TypeScript compiler not being able to independently specify jsx pragma and jsxFrag pragma.
```

Comment:
This, and related stuff, could be moved to the new "With the old JSX transform" section

Reply:
The fragment-related part was not yet re-introduced to that "With the old JSX transform" section

Reply:
My bad, should be good now.